gtk4/src/subclass/editable.rs

```diff
@@ -4,10 +4,68 @@ use super::widget::WidgetImpl;
 use crate::Editable;
 use glib::subclass::prelude::*;
 use glib::translate::*;
-use glib::{Cast, GString};
+use glib::{Cast, GString, ObjectExt, Quark};
 use libc::{c_char, c_int};
+use once_cell::sync::Lazy;
 
 pub trait EditableImpl: WidgetImpl {
+    fn properties(&self) -> &'static [String] {
+        static PROPS: Lazy<Vec<String>> = Lazy::new(|| {
+            vec![
+                "xalign".to_string(),
+                "width-chars".to_string(),
+                "text".to_string(),
+                "selection-bound".to_string(),
+                "max-width-chars".to_string(),
+                "editable".to_string(),
+                "cursor-position".to_string(),
+                "enable-undo".to_string(),
+            ]
+        });
+        PROPS.as_ref()
+    }
+
+    #[doc(alias = "gtk_editable_delegate_get_property")]
+    fn get_property(
+        &self,
+        editable: &Self::Type,
+        prop_id: usize,
+        pspec: &glib::ParamSpec,
+    ) -> Option<glib::Value> {
+        unsafe {
+            let mut value = glib::Value::uninitialized();
```

Comment:
`value` must be initialized to the target property type or it will cause critical warnings.

Reply:
```suggestion
            let mut value = glib::Value::from_type(pspec.get_value_type());
```